project/app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,208 @@
+    <string name="share">Compartir</string>
+    <string name="save">"Guardar"</string>
+    <string name="cancel">"Cancelar"</string>
+    <string name="preferencesCleanSessionEnabled">Limpiar sesión</string>
+    <string name="publish">Reportar</string>
+    <string name="preferencesPubExtendedDataSummary">Incluir datos adicionales en los reportes de ubicación</string>
+    <string name="preferencesPubExtendedData">Datos adicionales</string>
+    <string name="preferencesAutostartSummary">Arrancar cuando se enciende el dispositivo</string>
+    <string name="preferencesAutostart">Autoarranque</string>
+    <string name="preferencesLocatorAccuracyBackground">Locator background accuracy</string>
+    <string name="preferencesLocatorAccuracyBackgroundSummary">Precisión y consumo en segundo plano</string>
+    <string name="preferencesLocatorAccuracyForeground">Locator foreground accuracy</string>
+    <string name="preferencesLocatorAccuracyForegroundSummary">Precisión y consumo en primer plano</string>
+    <string name="preferencesRemoteCommand">Comandos remotos</string>
+    <string name="preferencesRemoteCommandSummary">Habilitar comandos remotos</string>
+    <string name="description">Descripción</string>
+    <string name="coordinates">Coordenadas</string>
+    <string name="longitude">Longitud</string>
+    <string name="latitude">Latitud</string>
+    <string name="radius">Radio</string>
+    <string name="addWaypoint">Añadir waypoint</string>
+    <string name="exportConfiguration">Exportar</string>
+    <string name="contactLocationUnknown">Ubicación no disponible</string>
+    <string name="menuContactNavigate">Navegar</string>
+    <string name="menuCurrentLoction">Ubicación actual</string>
+
+    <string name="contactDetailsAccuracy">Precisión</string>
+    <string name="contactDetailsReportedTime">Reportado</string>
+    <string name="contactDetailsTopic">Identificador</string>
+    <string name="waypointListPlaceholder">No hay regiones definidas</string>
+    <string name="waypointRemove">Eliminar</string>
+    <string name="importConfig">Importar</string>
+    <string name="tls" translatable="false">TLS</string>
+    <string name="preferencesSecurity">Seguridad</string>
+    <string name="accept">Aceptar</string>
+    <string name="preferencesIdentification">Identificación</string>
+    <string name="preferencesParameters">Parámetros</string>
+    <string name="preferencesKeepalive" translatable="false">Keepalive</string>
+    <string name="seconds">Segundos</string>
+    <string name="preferencesProfileId">Modo</string>
+
+
+    <string-array name="locatorAccuracy_readable">
+        <item>Alto rendimiento</item>
+        <item>Equilibrador</item>
+        <item>Bajo rendimiento</item>
+        <item>Apagado</item>
+    </string-array>
+
+
+    <string name="mode_mqtt_private_label">MQTT privada</string>
+    <string name="mode_mqtt_public_label">MQTT pública</string>
```

Comment:
@binarybucks this should exist any more I think

Reply:
`mode_mqtt_public_label` is no longer used. It can be purged from all strings.xml. 
The value of `mode_mqtt_private_label` can be changed to `MQTT` only in all strings.xml. Same for `mode_http_private_label`.

Reply:
@Kaysera don't worry about that (i.e. just ignore it); we'll do that once you're done.

Reply:
I'll remove them, don't worry